test/api/stripe_test.go

```diff
@@ -147,6 +151,134 @@ func testStripeCheckoutPOST(t *testing.T, at *test.AccountsTester) {
 	}
 }
 
+// testStripeCheckoutIDGET ensures that we can get the info for a checkout
+// session and act on it, i.e. promote the user, if needed.
+func testStripeCheckoutIDGET(t *testing.T, at *test.AccountsTester) {
+	name := test.DBNameForTest(t.Name())
+	// Create a test user.
+	r, _, err := at.UserPOST(name+"@siasky.net", name+"pass")
+	if err != nil {
+		t.Fatal(err)
+	}
+	c := test.ExtractCookie(r)
+	at.SetCookie(c)
+	defer func(c *http.Cookie) {
+		at.SetCookie(c)
+		_, _ = at.UserDELETE()
+	}(c)
+	u, _, err := at.UserGET()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Fixture values.
```

Comment:
maybe move it into constants on top of the file so it's reusable by other tests and it doesn't clutter the actual test